internal/xds/translator/authentication.go

```diff
@@ -195,29 +194,23 @@ func buildXdsUpstreamTLSSocket() (*corev3.TransportSocket, error) {
 }
 
 // patchRouteWithJwtConfig patches the provided route with a JWT PerRouteConfig, if the
-// route doesn't contain it. The listener is used to create the PerRouteConfig JWT
-// requirement.
-func patchRouteWithJwtConfig(route *routev3.Route, irRoute *ir.HTTPRoute, listener *listenerv3.Listener) error { //nolint:unparam
+// route doesn't contain it.
+func patchRouteWithJwtConfig(route *routev3.Route, irRoute *ir.HTTPRoute) error { //nolint:unparam
```

Comment:
why `unparam`?

Reply:
Have removed it. It was there earlier so somehow I missed that. Thanks for pointing out